widget/scroller.go

```diff
@@ -9,12 +9,16 @@ import (
 	"fyne.io/fyne/theme"
 )
 
-type scrollDirection int
+// ScrollDirection represents the directions in which a ScrollContainer can scroll its child content.
+type ScrollDirection int
 
 const (
-	scrollBoth = iota
-	scrollHorizontalOnly
-	scrollVerticalOnly
+	// ScrollBoth allows the ScrollContainer to scroll its child in both horizontal and vertical directions.
+	ScrollBoth ScrollDirection = iota
+	// ScrollHorizontalOnly allows the ScrollContainer to only scroll its child in the horizontal direction.
+	ScrollHorizontalOnly
+	// ScrollVerticalOnly allows the ScrollContainer to only scroll its child in the vertical direction.
+	ScrollVerticalOnly
```

Comment:
FYI: I think for constants that are mostly self-describing (which of course they should be ;)), a simple comment before the `const` block should be sufficient:

```go
// ScrollDirection represents the directions in which a ScrollContainer can scroll its child content.
type ScrollDirection int

// Constants for valid values of ScrollDirection.
const (
	ScrollBoth ScrollDirection = iota
	ScrollHorizontalOnly
	ScrollVerticalOnly
)
```

Reply:
Done